python/foglamp/services/core/service_registry/monitor.py

```diff
@@ -46,7 +46,7 @@ def __init__(self):
     async def _monitor_loop(self):
         """Main loop for the scheduler"""
         # check health of all micro-services every N seconds
-        _MAX_ATTEMPTS = 5
+        _MAX_ATTEMPTS = 15
```

Comment:
I believe its just too high 👎

Reply:
it is warranted to take care of network latency. I have experienced ping at 9th attempt even, hence this value to assure that service has really shutdown.